### ISSUE TITLE
Final Fantasy X/X-2 HD Remaster (359870)

### ISSUE THREAD

**Issue by nstgc** [ValveSoftware/Proton#555]
Log: https://gist.github.com/nstgc/58f25fc3e0439b658b6b1749fb2011ae
System: https://gist.github.com/nstgc/02457d7920931a3f8d69cc7b9fa41a2c
Unlike Sacred 2, FF X/X-2 works well in Linux.

**Comment by nstgc** [comment]
Sorry. Let me clarify: The game CTDs with Proton, but normally it runs fine.

**Comment by ryad-lindner** [comment]
Cannot launch FFX/X2 HD. Same problems as @bambinook : Stays on white screen and related processes have to be killed manually.
Config: Arch Linux, GTX 765M @ 396.51, Proton 3.7-4 beta

**Comment by rcgamer29** [comment]
I can get the launcher to launch and get the game to start but it freezes after about 10 seconds. If I use the PROTON_USE_WINED3D command there are graphical glitches but it will run. To get the launcher to work you have to change the prefix to xp so it will install .net. Then switch it back to Windows 7. No matter what the launcher will have to be killed through system monitor when you are done.

**Comment by nrdxp** [comment]
@rcgamer29 your solution of switching the prefix to xp to get the launcher to work is not working for me. This game will not launch at all no matter what I try. No white screen nothing. 

I'm using Proton 3.16.

**Comment by cuongttran** [comment]
I just installed FFX-X2 on Ubuntu 18.04 and have the same problem with @bambinook . The launcher come up, but when I select setting it just disappear. When select the game, it's stays on black screen.
I tried to add `PROTON_USE_WINED3D=1 %command%` to the game's launch options but the problem is still the same.

**Comment by nrdxp** [comment]
The issue is described more clearly on the dxvk project with a similar issue 
https://github.com/doitsujin/dxvk/issues/263

if you indeed switch the prefix to windows xp and install dotnet4 and allfonts, switch back to windows 7 and add 'PROTON_FORCE_LARGE_ADDRESS_AWARE=1 %command%' to the launch options the game will run with dxvk. There is a caveat because of the memory restrictions described in the thread above, you cannot use high graphical settings. I was able to get the game to launch with medium settings at 1080p with no crashing but high settings consistently crash the game.

Dxvk is definitely preferable as performance is much better than wined3d, and the black texture issues are gone. If your willing to accept the graphical limitations its the best way to play it right now.

**Comment by Glog78** [comment]
Sadly with LAA set the game crashes when summoning aeons (FFX) .. so even this LAA solution lets you only play the first little bit till yuna summon her first aeon. There is https://steamcommunity.com/groups/SpecialK_Mods/discussions/8/2741975115064718432/ but i sadly didn't get this one to run cause it has "memory fixes" v 0.6.6. So most likely PROTON_USE_WINED3D=1 is the best currently , sadly cause the rendering bugs destroyed the immersion for me too much.

**Comment by kisak-valve** [comment]
Final Fantasy X/X-2 HD Remaster (359870) Steam Input non-functional

Issue transferred from https://github.com/ValveSoftware/Proton/issues/3219.
@Rossco1337 posted on 2019-11-14T09:08:27:

# Compatibility Report
- Name of the game with compatibility issues: Final Fantasy X/X-2 HD Remaster
- Steam AppID of the game: 359870

## System Information
- GPU: GTX 1070 Ti
- Driver/LLVM version: nvidia-440.26
- Kernel version: 5.3.8.3
- Link to full system information report as [Gist](https://gist.github.com/Rossco1337/7860c4459caabd07b05b89f6b12c791f)
- Proton version: 4.11-8 (Also tested 4.19-GE)

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.


[steam-359870.log](https://github.com/ValveSoftware/Proton/files/3845476/steam-359870.log)


## Symptoms 
Steam Input is not working correctly in this game. Guide button/Big Picture can be navigated but none of my 3 controllers can interact with the game. Tested Switch Pro Controller, Dualshock 3 and Nyko Airflo (Generic Gamepad ID 124b:4d01), 

## Reproduction
- Launch Final Fantasy VII/another Proton game to verify that Steam Input is working in other Proton games.
- Launch Final Fantasy X and pause the intro cutscene with Esc.
- None of the buttons are working.

I don't have a Windows environment or any friends who own this game to verify that this is a Proton issue, but I can't find anyone online using Windows with this issue. If this isn't a Proton issue, please direct me to the correct bug tracker. Thanks.

**Comment by amatrelan** [comment]
# Compatibility Report
- Name of the game with compatibility issues: Final Fantasy X/X-2 HD Remaster
- Steam AppID of the game: 359870

## System Information
- GPU: NVIDIA GeForce RTX 2070 SUPER
- Driver/LLVM version: nvidia 440.59
- Kernel version: 5.5.7-1-MANJARO
- [System information](https://gist.github.com/Amatrelan/8841eae3eadaf141ad7a7b33ca6c4502)
- Proton version: Proton-5.2-GE-2, proton-5.0-3, proton-4.11-13

## I confirm:
- [x] that I haven't found an existing compatibility report for this game.
- [x] that I have checked whether there are updates for my system available.

[Proton-5.2-GE-2.steam-359870.log ](https://gist.github.com/Amatrelan/e0982e9cdbc5199df48bbc93b2ed569a)
[proton-5.0-3.steam-359870.log](https://gist.github.com/Amatrelan/ac4a63d0d4c527cfc2416e2838458810)
[proton-4.11-13.steam-359870.log](https://gist.github.com/Amatrelan/cbe7b3c1f5eaf0226090b6d025e4bdc8)  

## Symptoms 
After playing while (usually under 1 min in game) game freezes completely. After while noticed that there is about 30 processes but after it freezes all others dies than 2. 

## Reproduction
- Install Game
- Run it
- If does not crash on in opening, go into gameplay and play forward (Usually crashes quite fast in opening credit roll)

**Comment by kisak-valve** [comment]
Release Final Fantasy X/X-2 HD remaster as 64 bit or with Large Address Aware support

Issue transferred from https://github.com/ValveSoftware/Proton/issues/3832.
@gdonval posted on 2020-05-05T13:59:30:

- Game: Final Fantasy X/X-2 HD remaster
- Steam AppID of the game: 359870

## Problem

The problem is described at length in https://github.com/doitsujin/dxvk/issues/263: basically that 32-bit game runs out of Video RAM because of both inefficiencies and DXVK and different VRAM layout in Linux compared to Windows. See https://github.com/doitsujin/dxvk/issues/1318#issuecomment-574464662 (and full issue) for more context, noting that what applies to D9VK does apply to DXVK.

## Common mitigation does not work

Large Address Aware (LAA), now activated by default in Proton, is commonly used to increase memory limits in 32-bit applications. However it **does NOT work** in this game. Activation triggers a soft crash: the game mechanics is stuck on specific, reproducible, conditions necessary for the gameplay.

## Actions

The point of this bug request is to ask both:

- to invest in some work in DXVK's memory allocator to collect and defragment memory (long term solution for all 32 bit games out there that can't make use of DXVK);
- and kindly ask Square Enix to release that specific game in 64-bit (or at the very least, fix their pointers so that LAA becomes possible).


## System Information
- GPU: 5700XT (but internet reports with nvidia and other amd cards as well)
- Driver/LLVM version: Mesa 20.0
- Kernel version: 5.6.10
- Proton version: 5.0-7

## Compatibility report?

I don't consider this to be a compatibility report: I'm kindly asking you to contact Square Enix to see if they can fix their game for Proton in very specific ways.

**Comment by iWeaker** [comment]
I don't think it's a DXVK problem as mentioned in other reports, and the X-2 is constantly crashing with Wine version 5.0 / Proton5.0+, it is unplayable, also detect that PROTON_FORCE_LARGE_ADDRESS_AWARE has problems and is not functional in this title, so much more crashed when activated, i try version 4.11.13 and use this only PROTON_USE_D9VK = 1 PROTON_USE_WINED3D = 1% command% and it works great, it has graphic problems in the save menus but nothing serious, I could get to the ship and save the game, something that I can't do in Proton 5.0, because it crashes as soon as the first fight starts, even if I set those parameters.

**Comment by dowttie** [comment]
The game is currently mostly playable with PROTON_USE_WINED3D and PROTON_FORCE_LARGE_ADDRESS_AWARE=0
The graphics seem to start glitching out in some particular scenes (eg Yuna's introduction in FFX), but since the game is generous with autosaves you can just close the game and then reload the autosave after the glitched scene, then the game will play fine again.
WineD3D still isn't ideal though, some menu textures are shown as just white rectangles, and performance is spotty.

**Comment by NerosTie** [comment]
It looks like that the soft crash is fixed with Proton 5.13-4 + DXVK 1.7.3. Can someone confirm? I only tested with "Special Attack" at the beginning of the game (which was a source of the soft crash) and not summoning an aeon.

**Comment by NerosTie** [comment]
Ok, I played ~6 hours and I summoned my first eon: no crash. I think the game is now playable :)

**Comment by minecraft2048** [comment]
Can anyone confirm that Lulu overdrive works? I use PS4 controller and even when I already rotate the right analog speed as fast as possible, the overdrive gauge doesn't fill up

EDIT: Using left and right keys on keyboard also doesn't fill the gauge

**Comment by minecraft2048** [comment]
FFX2 always crashes at the first battle, but FFX is stable and I have played over 60 hours without any crashes. I'm using Proton 5.13-4 for both of them
[steam-359870.log](https://github.com/ValveSoftware/Proton/files/5790058/steam-359870.log)

 System info: https://gist.github.com/minecraft2048/c4f4d73a0e496168e80b1f12bf284d9d

EDIT: Proton Experimental doesn't crash at first battle. But, sometime it randomly segfaults and crash at battle (c0000005). This is very bad for this kind of game as we cannot save anywhere, only at save points
Logfile: [steam-359870.tar.gz](https://github.com/ValveSoftware/Proton/files/5824074/steam-359870.tar.gz)
Uncompressed file size is 49.7 MB, but most of the lines in the log is from NtQueryInformationThread and NtSetInformationThread

**Comment by enricotagliavini** [comment]
Using proton experimental since a couple of weeks and FFX works quite well, just very rare occasional crashes. FFX2 crashes fairly frequently, but it's a bit difficult to pinpoint how to reproduce in a reliable way. Sometimes it works for a couple of hours straight, sometimes it crashes every few minutes.

This is the log generate with PROTON_LOG=1 
[steam-359870-ffx2_crash.log.tar.gz](https://github.com/ValveSoftware/Proton/files/6135364/steam-359870-ffx2_crash.log.tar.gz)

I also tried the debug scripts generated with PROTON_DUMP_DEBUG_COMMANDS=1, but both winedbg* and gdb* scripts are failing to get a backtrace. It looks like they are tracing the launcher and not the game itself. I tried to attach to the wpid directly, but that fails with an error.

```
$ ps aux | grep -i ffx
enrico     58256  0.0  0.0 1793096 19972 ?       Ssl  15:52   0:00 /home/enrico/.local/share/Steam/steamapps/common/FINAL FANTASY FFX&FFX-2 HD Remaster/FFX&X-2_LAUNCHER.exe
enrico     87434  0.6  0.0   3512   892 ?        S    19:57   0:00 /usr/bin/bwrap --args 35 /run/pressure-vessel/pv-from-host/bin/pressure-vessel-adverb --generate-locales --fd 12 --exit-with-parent --subreaper --shell=none --terminal=none -- /home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/proton waitforexitandrun /home/enrico/.local/share/Steam/steamapps/common/FINAL FANTASY FFX&FFX-2 HD Remaster/FFX&X-2_LAUNCHER.exe
enrico     87839  0.1  0.0  52640 27140 ?        Ss   19:57   0:00 /run/pressure-vessel/pv-from-host/bin/pressure-vessel-adverb --generate-locales --fd 12 --exit-with-parent --subreaper --shell=none --terminal=none -- /home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/proton waitforexitandrun /home/enrico/.local/share/Steam/steamapps/common/FINAL FANTASY FFX&FFX-2 HD Remaster/FFX&X-2_LAUNCHER.exe
enrico     87863  0.4  0.0  41612 16420 ?        S    19:57   0:00 python3 /home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/proton waitforexitandrun /home/enrico/.local/share/Steam/steamapps/common/FINAL FANTASY FFX&FFX-2 HD Remaster/FFX&X-2_LAUNCHER.exe
enrico     87866  1.5  0.0 1823544 27668 ?       Sl   19:57   0:00 steam /home/enrico/.local/share/Steam/steamapps/common/FINAL FANTASY FFX&FFX-2 HD Remaster/FFX&X-2_LAUNCHER.exe
enrico     87994 42.5  1.0 3424192 338140 ?      Ssl  19:57   0:07 FFX-2.exe    <---- this is the game
enrico     88207  0.0  0.0 221568   796 pts/6    S+   19:57   0:00 grep --color=auto -i ffx



$ bash -x ./gdb_attach FFX-2.exe
+ '[' -z '' ']'
+ case "$-" in
+ __lmod_vx=x
+ '[' -n x ']'
+ set +x
Shell debugging temporarily silenced: export LMOD_SH_DBG_ON=1 for this output (/usr/share/lmod/lmod/init/bash)
Shell debugging restarted
+ unset __lmod_vx
+ cd '/home/enrico/.local/share/Steam/steamapps/common/FINAL FANTASY FFX&FFX-2 HD Remaster'
+ EXE_NAME=FFX-2.exe
++ /tmp/proton_enrico/winedbg --command 'info process'
++ grep -i FFX-2.exe
++ cut -f2 '-d '
++ tr -d 0
+ WPID_HEX=11
+ '[' -z 11 ']'
++ printf %d 0x11
+ WPID_DEC=17
+ PATH='/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/bin/:/usr/bin:/bin'
+ TERM=xterm
+ WINEDEBUG=-all
+ WINEDLLPATH='/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/lib64//wine:/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/lib//wine'
+ LD_LIBRARY_PATH='/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/lib64/:/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/lib/:/overrides/lib/x86_64-linux-gnu:/overrides/lib/i386-linux-gnu'
+ WINEPREFIX=/home/enrico/.local/share/Steam/steamapps/compatdata/359870/pfx/
+ WINEESYNC=1
+ WINEFSYNC=1
+ SteamGameId=359870
+ SteamAppId=359870
+ WINEDLLOVERRIDES='steam.exe=b;dotnetfx35.exe=b;dxvk_config=n;d3d11=n;d3d10=n;d3d10core=n;d3d10_1=n;d3d9=n'
+ STEAM_COMPAT_CLIENT_INSTALL_PATH=/home/enrico/.local/share/Steam
+ WINE_LARGE_ADDRESS_AWARE=1
+ GST_PLUGIN_SYSTEM_PATH_1_0='/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/lib64/gstreamer-1.0:/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/lib/gstreamer-1.0'
+ WINE_GST_REGISTRY_DIR=/home/enrico/.local/share/Steam/steamapps/compatdata/359870/gstreamer-1.0/
+ MEDIACONV_AUDIO_DUMP_FILE=/home/enrico/.local/share/Steam/steamapps/shadercache/359870/fozmediav1/audio.foz
+ MEDIACONV_AUDIO_TRANSCODED_FILE=/home/enrico/.local/share/Steam/steamapps/shadercache/359870/swarm/transcoded_audio.foz
+ MEDIACONV_VIDEO_DUMP_FILE=/home/enrico/.local/share/Steam/steamapps/shadercache/359870/fozmediav1/video.foz
+ MEDIACONV_VIDEO_TRANSCODED_FILE=/home/enrico/.local/share/Steam/steamapps/shadercache/359870/swarm/transcoded_video.foz
+ '/home/enrico/.local/share/Steam/steamapps/common/Proton - Experimental/dist/bin/wine' winedbg --gdb 17
Can't attach process 0011: error 87
Couldn't start process '17 '
Usage:
        winedbg [ [ --gdb ] [ <prog-name> [ <prog-args> ] | <num> | <file.mdmp> | --help ]
```

This is because the process PID ends with a zero which is removed by the ```tr -d '0'```. Editing the script and replacing the tr with ```sed 's/0*//'``` seems to fix the issue. If I can get a trace, I'll upload it.

**Comment by enricotagliavini** [comment]
No backtrace unfortunately

```
Thread 32 "010c" received signal SIGSEGV, Segmentation fault.
[Switching to Thread 268]
0x006a8bd9 in ?? ()
Wine-gdb> bt
#0  0x006a8bd9 in ?? ()
#1  0x006bc2b8 in ?? ()
#2  0x006a5e35 in ?? ()
#3  0x00605d66 in ?? ()
#4  0x00607073 in ?? ()
#5  0x0042f520 in ?? ()
#6  0x00abed7e in ?? ()
#7  0x0087dc42 in ?? ()
#8  0x7b6470f1 in ?? ()
#9  0x7b647738 in ?? ()
#10 0x7b647102 in ?? ()
#11 0x00000000 in ?? ()
```

**Comment by enricotagliavini** [comment]
```PROTON_USE_WINED3D=1 PROTON_FORCE_LARGE_ADDRESS_AWARE=0 %command%```

This made the game stable, no more crashes whatsoever for FFX-2. Performance is a bit spotty in rare cases and load / save menu doesn't render correctly, but is usable. Enjoyable overall this way.

**Comment by nstgc** [comment]
> `PROTON_USE_WINED3D=1 PROTON_FORCE_LARGE_ADDRESS_AWARE=0 %command%`
> 
> This made the game stable, no more crashes whatsoever for FFX-2. Performance is a bit spotty in rare cases and load / save menu doesn't render correctly, but is usable. Enjoyable overall this way.

Could you share your system information and which version of Proton you're using? Many of us, myself included, have tried that before, but without the desired effect.

**Comment by enricotagliavini** [comment]
> Could you share your system information and which version of Proton you're using? Many of us, myself included, have tried that before, but without the desired effect.

Absolutely. I'm running proton experimental, from the log it says

```
Proton: 1615470153 experimental-5.13-20210311
```

On Fedora 33. Following are the system information as detected by Steam
[steam_runtime_info.txt](https://github.com/ValveSoftware/Proton/files/6175754/steam_runtime_info.txt)

Hit me if you have any other question

**Comment by NerosTie** [comment]
I experience the same random crashes as @enricotagliavini.

Version of Proton: experimental-5.13-20210311
GPU: AMD RX 580 (Mesa 21.0.1)

This is my log:

[steam-359870.tar.gz](https://github.com/ValveSoftware/Proton/files/6239979/steam-359870.tar.gz)

It could be an issue with DXVK, but to make a trace will be more difficult and the result file heavy.

**Comment by alex9k1** [comment]
> Can anyone confirm that Lulu overdrive works? I use PS4 controller and even when I already rotate the right analog speed as fast as possible, the overdrive gauge doesn't fill up
> 
> EDIT: Using left and right keys on keyboard also doesn't fill the gauge

Wanted to confirm, that this is still an issue with Proton 6.3-4 and Experimental. Quite annoying :(

**Comment by cybik** [comment]
For the record, FFX Audio still doesn't work on either latest Stable or latest GE, and I had a crash in the intro just when Tidus comes into the screen (without audio).

**Comment by bjlindst** [comment]
I had similar random freezes and crashes, in FFX-2. FFX ran great throughout the whole game with just the default settings, except for Lulu's overdrive. In FFX-2, and everything on default, the video would Freeze just after "Final fantasy X-2 project 2002-2004" appeared, but the music was playing in the background. No keyboard input did anything.

I tried some things above, and sometimes I would get into the game and do a couple of battles before it crashed.

Now it seems to be running fine however (at least first 30 min of gameplay), by doing the following:
* Disable Steam overlay in-game**
* PROTON_USE_WINED3D=1 PROTON_FORCE_LARGE_ADDRESS_AWARE=0 %command% in the launch options
* Force the use of Proton 6.3-6 in the compatibility tab of the game settings (I tried some others but then it would crash)
* Setting graphics to lowest in the settings menu before launching the game (where you choose between FFX / FFX-2)**

I am running Manjaro with kernel 5.13.11-1, Ryzen 5600g, 64 GB RAM, no dedicated GPU.

** these two settings I did not try with all combinations of different proton versions, so perhaps they are not important or perhaps they make another proton version work

**Comment by DistantThunder** [comment]
Launching the game on Arch Linux 5.13.12-zen1-1-zen  / Proton 6.3-6, I reproduce the problems above:

* Launching with intro video playing but no sound: 
[steam-359870.log.tar.gz](https://github.com/ValveSoftware/Proton/files/7117982/steam-359870.log.tar.gz)
* Successful launch, no graphics freezing in the intro video & sound fine
[steam-359870-success.log.tar.gz](https://github.com/ValveSoftware/Proton/files/7118045/steam-359870-success.log.tar.gz)
* After a gameplay session and a suspend/resume cycle, the game won't start with sound:
[steam-359870-failnosound.log.tar.gz](https://github.com/ValveSoftware/Proton/files/7122662/steam-359870-failnosound.log.tar.gz)

**Comment by RyuzakiKK** [comment]
It seems like the current Proton Experimental (experimental-6.3-20210924) introduced a regression.

## Steps to reproduce

- Launch FINAL FANTASY X/X-2 HD Remaster
- In the first menu go to the game settings
- Press the first drop down menu to attempt to change the game language (the default is Japanese)

## Expected behavior

A drop down list appears to let you choose the language

## Actual behavior

It crashes with the following popup:
![Screenshot from 2021-09-29 21-47-21](https://user-images.githubusercontent.com/3227973/135338221-1f2070e9-b31d-4edc-b4de-e4115c1b9b29.png)

"Details" contains:
```
See the end of this message for details on invoking \njust-in-time (JIT) debugging instead of this dialog box.\n\n************** Exception Text **************\nSystem.Runtime.InteropServices.MarshalDirectiveException: Marshalling of type object is not implemented
  at (wrapper managed-to-native) System.Windows.Forms.UnsafeNativeMethods.UiaRaiseAutomationPropertyChangedEvent(System.Windows.Forms.UnsafeNativeMethods/IRawElementProviderSimple,int,object,object)
  at System.Windows.Forms.AccessibleObject.RaiseAutomationPropertyChangedEvent (System.Int32 propertyId, System.Object oldValue, System.Object newValue) [0x0000b] in <cee14d93d65d43acb1c93793b46e63c3>:0 
  at (wrapper remoting-invoke-with-check) System.Windows.Forms.AccessibleObject.RaiseAutomationPropertyChangedEvent(int,object,object)
  at System.Windows.Forms.ComboBox.OnDropDown (System.EventArgs e) [0x0002e] in <cee14d93d65d43acb1c93793b46e63c3>:0 
  at System.Windows.Forms.ComboBox.WmReflectCommand (System.Windows.Forms.Message& m) [0x000b4] in <cee14d93d65d43acb1c93793b46e63c3>:0 
  at System.Windows.Forms.ComboBox.WndProc (System.Windows.Forms.Message& m) [0x00202] in <cee14d93d65d43acb1c93793b46e63c3>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.OnMessage (System.Windows.Forms.Message& m) [0x00001] in <cee14d93d65d43acb1c93793b46e63c3>:0 
  at System.Windows.Forms.Control+ControlNativeWindow.WndProc (System.Windows.Forms.Message& m) [0x000b3] in <cee14d93d65d43acb1c93793b46e63c3>:0 
  at System.Windows.Forms.NativeWindow.Callback (System.Windows.Forms.Message& m) [0x00025] in <cee14d93d65d43acb1c93793b46e63c3>:0 

\n************** Loaded Assemblies **************\nmscorlib\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/4.5/mscorlib.dll\n----------------------------------------\nGameLauncher\n    Assembly Version: 1.0.0.0\n    Win32 Version: 1.0.0.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/FINAL%20FANTASY%20FFX&FFX-2%20HD%20Remaster/FFX&X-2_LAUNCHER.exe\n----------------------------------------\nSystem.Drawing\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/System.Drawing/4.0.0.0__b03f5f7f11d50a3a/System.Drawing.dll\n----------------------------------------\nSystem\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/System/4.0.0.0__b77a5c561934e089/System.dll\n----------------------------------------\nSystem.Windows.Forms\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/System.Windows.Forms/4.0.0.0__b77a5c561934e089/System.Windows.Forms.dll\n----------------------------------------\nAccessibility\n    Assembly Version: 4.0.0.0\n    Win32 Version: \n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/Accessibility/4.0.0.0__b03f5f7f11d50a3a/Accessibility.dll\n----------------------------------------\nSystem.Core\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/System.Core/4.0.0.0__b77a5c561934e089/System.Core.dll\n----------------------------------------\nMono.Security\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.0.0.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/Mono.Security/4.0.0.0__0738eb9f132ed756/Mono.Security.dll\n----------------------------------------\nSystem.Xml\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/System.Xml/4.0.0.0__b77a5c561934e089/System.Xml.dll\n----------------------------------------\nSystem.Configuration\n    Assembly Version: 4.0.0.0\n    Win32 Version: 4.6.57.0\n    CodeBase: file:///Z:/run/media/nvme_data/Steam%20Games%202/steamapps/common/Proton%20-%20Experimental/files/share/wine/mono/wine-mono-6.4.0/lib/mono/gac/System.Configuration/4.0.0.0__b03f5f7f11d50a3a/System.Configuration.dll\n----------------------------------------\n\n************** JIT Debugging **************\n
```

(sorry for the unformatted output, but that's exactly what the popup window contains)

## Additional info

If I switch to Proton 6.3-6, the language list is displayed correctly.

Let me know if you need more information.

**Comment by madewokherd** [comment]
Thanks, that should be fixed in the develop branch of wine-mono, I'm working on getting it into a release so we can update Proton.

**Comment by ThisNekoGuy** [comment]
I mean... getting the launcher working an all is great but there really needs to be a real solution for the muted audio problem the game has in Proton too :/
No audio is the same as an unplayable game for a lot of people and the "just restart the game" method doesn't always work and just straight up doesn't work in some Proton versions (in my experience, I've never got it to work with audio)

**Comment by RyuzakiKK** [comment]
> Thanks, that should be fixed in the develop branch of wine-mono, I'm working on getting it into a release so we can update Proton.

I confirm that with the latest build `1633981594 experimental-6.3-20211011b` this issue has been fixed.

Now the only problem left that I have is that sometimes "Final Fantasy X" doesn't have any audio (but the launcher does). When this happens, realunching the game is usually enough to fix it.

**Comment by jtrees** [comment]
~~I can no longer get in game. Last time I tried is a few months back. Now I get the Square Enix logo, the save notice with the running chocobo and then a black screen. Rarely (about 10% of the time) do I hear the To Zanarkand background music while the black screen is there.~~

- ~~Proton 6.3-8 (I also tried Experimental)~~
- ~~NixOS 21.11~~
- ~~Linux 5.10.84~~
- ~~Nvidia (proprietary) 495.44~~

~~Let me know if you're interested in logs or other information.~~

EDIT: Sorry, I had a bad NixOS config. It's working now.

**Comment by ShadwDrgn** [comment]
Valve support says I should submit my bug for this game here. My steam deck lists this game as "playable" but it crashes on load before I ever get a menu. it seems like if i spend an hour or two just launching the game over and over and over it WILL finally load very very rarely and I can get to gameplay and play it once i do, but it's almost impossible to get that far. I'm not sure how to generate a crash report with my steam deck

**Comment by Eoin-ONeill-Yokai** [comment]
@enricotagliavini

> Replying to https://github.com/ValveSoftware/Proton/issues/555#issuecomment-798769427

Played this on the deck recently and this was my experience. Game worked perfectly for the entirety of X, but can occasionally crash on X-2. It's not entirely random, because the crashes have only happened to me (so far) on battle transitions leading into battle (breaking glass screen effect, generally fully plays and then it's a black screen.) No game audio or inputs are responsive at that point.

**Comment by Figuera** [comment]
I was able to finished playing FFX under Proton and played the starting mission of FFX-2. However, today I tried to play the game again and it doesn't even start (the launcher works but nothing else).

The worst part is that there is not really any debugging message for me to work on, the proton log show some warning messages but no clear error:

[steam-359870.log](https://github.com/ValveSoftware/Proton/files/9848638/steam-359870.log)

I wonder if this is happening to anyone else.

Using Arch Linux under Wayland (Sway).

**Comment by kisak-valve** [comment]
Hello @Figuera, these look like some lines of interest from your log:
```
Missing interface adapter for SteamUser023 User
err:steamclient:create_win_interface Don't recognize interface name: SteamUser022
Missing interface adapter for SteamUtils012 Utils
Missing interface adapter for SteamUtils011 Utils
```

**Comment by ivyl** [comment]
@kisak-valve I don't think that's the problem. Those missing interfaces are not even a part of current Steamworks SDK 155.

@Figuera The game works for me with current experimental. Looks like you are using a very old, non-official Proton. From your log:

```
Proton: 1643512896 7.1-GE-2
```

You should not report issues with those here. Always check with Proton proper before posting.

**Comment by Figuera** [comment]
@ivyl

I was also skeptical of @kisak-valve comments and I don't know what the problem actually was but after a steam update (basically yesterday update) the game started to work again.

So maybe the lines were correct and it was a steam related problem?

I am still using 7.1-GE-2, it always worked fine, I wouldn't call it very old and I tried to update it when the problem started but it didn't change anything.

**Comment by ivyl** [comment]
@Figuera are you on Steam Client beta? https://steamcommunity.com/groups/SteamClientBeta/announcements/detail/3428954061821341725 might have been this.

**Comment by Figuera** [comment]
I am using Arch Linux's default steam version. I am not sure if it is based on beta or not (I assume it does not). But that error description sounds exactly like what I was experiencing.

**Comment by kisak-valve** [comment]
Final Fantasy X/X-2 HD Remaster crashes on launch

Issue transferred from https://github.com/ValveSoftware/Proton/issues/6921.
@kurtbixby posted on 2023-07-11T06:21:33:

# Compatibility Report
- Name of the game with compatibility issues: Final Fantasy X/X-2 HD Remaster
- Steam AppID of the game: 359870

## System Information
- GPU: Steam Deck
- Driver/LLVM version: Mesa 22.2.0
- Kernel version: 5.13.0
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: 7.0-6

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Proton Log
https://gist.github.com/kurtbixby/8c0366f76b68f65f3de5d90daf01f54d

## Symptoms 
Final Fantasy X-2 crashes on boot

## Reproduction
1) Start Final Fantasy X/X-2
2) Select Final Fantasy X-2 from the launcher
3) Game will crash after the autosave warning
***
@kurtbixby commented on 2023-07-11T07:25:09:

This appears to be a regression. The game launches successfully on Proton 6.3-8.

**Comment by runar-work** [comment]
I was unable to reproduce a crash with the same mesa, kernel and proton version, so not sure what this could be.

**Comment by alasky17** [comment]
@kurtbixby Unfortunately, we are not seeing this crash either.  Could you try a few things to see if we can get to the bottom of this? 
1) Could you get a log on Proton - Experimental? 
2) After getting the crash, could you save the output of "coredumpctl" and upload that too?
3) Could you try moving/renaming the prefix (located by default in ~/.steam/steam/steamapps/compatdata/359870) and seeing if that fixes the problem.  If so, we would be interested to look at the broken prefix to see if we can diagnose what went wrong.
4) Do you already have a save file from this game? If so, could you upload it in case there is something in your save file that is causing the crash?
5) I'm assuming that you are using SteamOS stable.  If you are using a different SteamOS branch (or anything else "not standard) could you let us know?

**Comment by kurtbixby** [comment]
I think it was an issue relating to my save file. I was on an older version of Proton for a bit, where the game didn't crash, and after progressing in the game some, the game no longer crashed on the latest Proton. I don't have my problematic save anywhere, so unfortunately I can't help anymore.

**Comment by alasky17** [comment]
@kurtbixby Thank you for following up!  If you happen to run into this again, I'd greatly appreciate it if you could save the save file before continuing past the stuck point so that we can investigate :) Of course, I realize that is unlikely 😅

**Comment by kurtbixby** [comment]
While I don't have the save file, I have run into another issue.

# Compatibility Report
- Name of the game with compatibility issues: Final Fantasy X/X-2 HD Remaster
- Steam AppID of the game: 359870

## System Information
- GPU: Steam Deck
- Video driver version: Mesa 22.2.0
- Kernel version: 5.13.0
- Link to full system information report as [Gist](https://gist.github.com/):
- Proton version: 8.0-3

# Proton Log
https://gist.github.com/kurtbixby/fd812b17ca7420b7b71bb870228b714e

## I confirm:
- [X] that I haven't found an existing compatibility report for this game.
- [X] that I have checked whether there are updates for my system available.

## Symptoms

Game frequently crashes just before the glass breaking animation when entering a battle

## Reproduction

1. Load my autosave
2. Head up the stairs and run around the field until a battle starts
3. The game will often crash when entering one of these battles

I can attach my save directory if you'd like to test for yourselves.
[FINAL FANTASY X&X-2 HD Remaster.zip](https://github.com/ValveSoftware/Proton/files/12222725/FINAL.FANTASY.X.X-2.HD.Remaster.zip)

**Comment by alasky17** [comment]
@kurtbixby Did you/could you check to see if this crash happens on 6.3-8 too?

**Comment by kurtbixby** [comment]
I'll play more of the game on 6.3-8. I haven't forced a crash after about 10 battles, so the crash at least appears to be less frequent on 6.3-8.

**Comment by kurtbixby** [comment]
I've been playing on 7.0-6 and while the crashes are much less frequent, there are still occasional crashes when entering battle and occasionally during battle. I've had logging turned on and have a few crash logs that are over 100MBs.

I haven't been playing on 6.3-8 because that version of Proton doesn't provide the correct mapping for the Steam Deck's controls.

[X2Logs.zip](https://github.com/ValveSoftware/Proton/files/12255506/X2Logs.zip)

**Comment by jansalleine** [comment]
> Replying to https://github.com/ValveSoftware/Proton/issues/555#issuecomment-1630743654

I had the same crashing issue (on the Steam Deck), but found a solution:
The game cloud save will save the game resolution settings and fall back to 640 * 480 on Steam Deck if I did set a higher resolution at my desktop PC before that the Steam Deck doesn't support (2560 * 1440). That leads to crashing in Proton 8.0-3 (and Proton 7.0-6), but for some reason not in 6.3-8. 

If I go to the settings in the launcher and set the resolution to 1280 * 800 before playing on my Steam Deck it doesn't crash on Proton 8.0-3. The other way around doesn't seem to be a problem here, but I also started settings my desktop resolution again when I continue playing on my desktop.

The issue exists in FF X and FF X-2, but for some reason Proton 8.0-3 managed to start FF X after a few tries when crashes happened after the autosave warning while FF X-2 always crashed.

With the correct resolution set I can play FF X-2 on both Steam Deck and desktop with Proton 8.0-3 without problems or crashes.

**Comment by ThisNekoGuy** [comment]
@kisak-valve (Steam Deck report)

# Compatibility Report
- Name of the game with compatibility issues:  Final Fantasy X/X-2 HD Remaster
- Steam AppID of the game: 359870

## System Information
- GPU: AMD Custom GPU 0405 (vangogh, LLVM 15.0.7, DRM 3.54)
- Video driver version: 4.6 (Compatibility Profile) Mesa 23.1.3 (git-58f7632462)
- Kernel version: 6.1.43-valve1-1-neptune-61
- Link to full system information report: [Gist](https://gist.github.com/ThisNekoGuy/24d1fde66973523c1c5a86f793df87f6)
- Proton version: 8.0-3c

## I confirm:
- [x] that I have checked whether there are updates for my system available.

**Proton Log:** [steam-359870.zip](https://github.com/ValveSoftware/Proton/files/12519449/steam-359870.zip)


## Symptoms 
Attempting to use a specific character's "Overdrive" attack refuses to take any user inputs when prompted by the game to do so; the proton log seems to reference this, so maybe this information helps? (I don't know)
```
61840.605:017c:0180:trace:loaddll:free_modref Unloaded module L"Z:\\home\\deck\\.local\\share\\Steam\\steamapps\\common\\FINAL FANTASY FFX&FFX-2 HD Remaster\\magicFiles\\FFX\\magic_0251.dll" : native
```
Not being able to use an attack like this is quite frustrating :v

## Reproduction

1. Launch the game
2. Play the game until you have Lulu in your party
3. Fill her "overdrive" gauge to max after having her engage in combat enough
4. Attempt to use her "Overdrive" attack (properly) by rotating the right thumb-stick when you activate it

**Comment by alasky17** [comment]
@ThisNekoGuy Would you mind providing a save file so that we can investigate this bug quicker?

**Comment by ThisNekoGuy** [comment]
@alasky17 Here you go
[FFX Saves.tar.gz](https://github.com/ValveSoftware/Proton/files/12542841/FFX.Saves.tar.gz)
These saves are right before a boss fight and the affected character already has the overdrive gauge filled, so all you have to do is just switch her into combat (Left Bumper) and use the attack (left on the D-pad, select it along with the desired element, and you'll be prompted to rotate the right thumb-stick while a timer counts down)

**Comment by alasky17** [comment]
> > Replying to [#555 (comment)](https://github.com/ValveSoftware/Proton/issues/555#issuecomment-1630743654)
> 
> I had the same crashing issue (on the Steam Deck), but found a solution: The game cloud save will save the game resolution settings and fall back to 640 * 480 on Steam Deck if I did set a higher resolution at my desktop PC before that the Steam Deck doesn't support (2560 * 1440). That leads to crashing in Proton 8.0-3 (and Proton 7.0-6), but for some reason not in 6.3-8.
> 
@jansalleine I'm not seeing the crash with the resolution in the launcher set to 640x480 on the Steam Deck ... at what point in-game are you seeing the crash? I was looking at the first FFX-2 option and not "last mission".

Also - do you have an external monitor plugged into your deck while you are seeing this crash?

**Comment by jansalleine** [comment]
@alasky17 I assume this only happens when the resolution was set to a value that is not available on Deck on the Desktop PC. So the config "shows" 640 * 480 on the Deck, but somewhere saved is still 2560 * 1440 that is only overwritten once you save the config on the Deck again.

**Comment by biomagnet** [comment]
Unable to play the X-2 portion of the FFX/X-2 HD Remaster on Steamdeck. I have tried on Proton Experimental, 8.0-4 and 6.3-8. With proper resolution settings, the game will load and runs beautifully, but any battle becomes a ticking time-bomb because of the transitions; sometimes the swap between overworld and battle will just hard crash back to my library.

It can take some playing to make it happen, but it will eventually. So far, the furthest I've gotten without crashing has been racing Leblanc, Logos and Ormi to the top of Gagazet. If i reload and continue, I can sometimes get past it to earn the Black Mage Dressphere and move into another map, etc., but it will always crashes at some point.

**Comment by ProfBlack** [comment]
Like others I’m getting random crashes in X-2.
Have tried with Proton 6.3-8 as well as the standard Proton build as of October 9th 2023. 6.3-8 seems to be more stable but eventually the game will crash. Every time has been at the beginning or very near the beginning of a battle.

**Comment by Denmark14** [comment]
I'm encountering the same issue with the X-2 portion of the FFX-2 HD Remaster on my Steamdeck. I've attempted to play it on Proton Experimental, 8.0-4, and 6.3-8. Just like the above, when I configure the resolution settings correctly, the game launches and runs smoothly. However, battles in the game have become a problem due to transition issues. Occasionally, the transition between the overworld and battles results in a hard crash. In the rare occurrence that I do get past a battle transition, the game will hard crash after about 30 minutes of gameplay to some other form of scene transition.

**Comment by alasky17** [comment]
@Denmark14 @ProfBlack @biomagnet The most recent experimental release contains a fix for the crashing during combat.  Please let me know if you are still seeing the intermittent crashing.  It also has a fix for the resolution mismatch when cloud saves carry over, so you should be able to bounce between the deck and PC without issue now :)

**Comment by biomagnet** [comment]
Yo!!! I've been playing for hours now doing Creature Creator, which is nothing _but_ battles, and haven't had a single crash!

Thanks so much for this fix. I'm over the moon!

**Comment by hilts-vaughan** [comment]
I can confirm I've been playing for a couple hours without a crash now as well on experimental.

Could I ask what commit fixes this? Always love reading about these things.

**Comment by alasky17** [comment]
@hilts-vaughan Yay!!! Glad to hear it :) Interestingly, the game was running into bad addresses due to fact that LAA (Large Address Aware) is enabled by default in Proton.  It assumes addresses are within 2GB space and was crashing when it got an address that wasn't.  The commit is     "proton: Disable forced LAA for Final Fantsy X/X-2."

**Comment by tzui20** [comment]
Hi! I tried this morning with Proton experimental - seemed to work really well but still got a crash when transitioning to a battle :/ - unless I missed to adjust anything else? (resolution, etc?)

Thank you!

**Comment by alasky17** [comment]
@tzui20 Could you get a log of the crash? You can get a log by adding "PROTON_LOG=1 %command%" to launch options, and the log will show up in ~/steam-359870.log.

**Comment by piregg** [comment]
@alasky17 

[steam-359870.log](https://github.com/ValveSoftware/Proton/files/13627681/steam-359870.log)
I'm trying to play final fantasy x and it would crash straight away, perhaps to do with the movie scene?
I then switched to proton GE which worked for around 2 hours. Now I'm getting crashes again at a certain part. I have tried proton experimental and several other official protons as of 10th Dec 2023. I will try to add logs if I can figure out how 

edit, log attached. I had 2 crashes here almost at the exact same spot.

**Comment by alasky17** [comment]
@piregg Unfortunately the crash you are seeing now looks like the result of the first attempt to fix the crash during battles etc.  This has been reverted in the Proton Experimental bleeding-edge branch and replaced with a different hack that we hope will fix the crashing during battle without causing the game to run out of memory (which is what was happening with the original hack).  If folks have time to test the new version of this fix with the bleeding-edge beta branch of Proton Experimental, that would be greatly appreciated :)

**Comment by piregg** [comment]
@alasky17 wow thanks for the rapid change. I figured out how to apply the bleeding edge branch of Proton Experimental. I then loaded up FFX and the Save file I had at a particular section (that crashed in the same spot several times previously) let me through with no problem!
I then continued to play for around 2 hours and I had no problems whats so ever! I will continue to play this week and respond back in a week or so. I've leave logging on in case in crashes again.

I haven't tested FFX-2 - which is a separate game but part of the same package. Perhaps someone else could test this if they read this message?

I'm new to all this. What is the process for getting this fix (if it proves to be a success) into some kind of main or permanent branch that is auto-applied to this game in future? I imagine there are other players out there who are suffering these crashes due to the game being from a very popular franchise.

Thanks again for your rapid help!

**Comment by alasky17** [comment]
@piregg Coincidently, we were running into basically the same problem in our testing on Monday, which is part of how the problem got fixed so quickly :D The timing of your comment was pretty funny actually as we were just realizing the problem ourselves :)  The process is that this fix will get shipped in the regular (non-bleeding-edge) experimental branch as soon as we do another release (generally within 1-2 weeks) and then in this particular case it will also be shipped with the next 8.0-x stable release.  We are actually in the RC phase for 8.0-5, so if you would like to test and make sure the fix works as well with 8.0-5 as with experimental, you can select the "release-candidate" beta channel on Proton 8.0 (just like you did for bleeding-edge experimental).  Thank you for testing and for the feedback!

**Comment by piregg** [comment]
@alasky17 - I have played FFX for at least another 5 hours and I've had no crashes or problems of any kind. This is still using the bleeding edge branch. I dare to say I think this game is fully fixed! 
I haven't played FFX-2 yet (part of the same package) so I can't comment on that though. 
I will now switch over to 8.05 with Release-Candidate branch selected.

**Comment by ghost** [comment]
**How I fixed the Problem** with my Game Crashing. First I verified my files, then I checked which proton version I was running, I checked that I was running experimental. I went into the game settings and made sure the resolution was set to 1280x800. I launched the game, and seleted the game I wanted to play,. Before starting a new game I opened the game settings by hitting esc with my keyboard. I then went to the games settings and lowered all of the settings to low. Make sure resolution is set to 1280x800 again in the game settings. set antialising to 1x. I hope this helps someone else!

**Comment by Wedge009** [comment]
About a week ago I tried both X and X-2 and even the Eternal Calm bridging story segment and all of those seem to crash-to-desktop within a minute. Using vanilla Proton 9.0-2, 9.0-3 is current at time of writing. A surprising experience contrasting with my time with other 3D FF games, but I understand X/X-2 is unique in the series with its use of PhyreEngine.

If reducing graphical settings is necessary for stability, then probably not worth playing via Proton for now.

**Comment by simifor** [comment]
@Wedge009 Could you try to set proton experimental for this game? I did some testing with it and it worked well.

**Comment by Wedge009** [comment]
I'm not playing it right now, was just giving a report on current status. If it working in experimental that's usually a good sign it will work in the next releases.

**Comment by Neverlight-Jyria** [comment]
Suffering the same issue here, tried proton 6.0-7 too witch most people say it will make the game work but so fat I've tried experimental, 9, 8 ,7 and 6.x versions and they all seem to crash in the startup cutscene sadly

**Comment by simifor** [comment]
When I tested this last month I saw the game crash with proton stable
(9.0-3 at that point), but it was working fine with proton 8 and proton
experimental. Tried it just now on experimental and I could launch it 5
times in a row without any sort of crash, advanced a bit through the game
without issue as well.

Could you do the following:
-Switch proton version to experimental
-Add "PROTON_LOG=1 %command%" to the game's launch parameters, without the
quotes
-Upload the resulting log, you'll fin a file named "steam-359870.log" in
your home folder. Compress it if necessary.

**Comment by Wedge009** [comment]
Not sure what improvements from Experimental relevant to this game - if any - has been back-ported to Proton 9.0-4, but I just tried to view the Eternal Calm story and the scene froze at the second line, where Wakka calls out 'Yuna, it's time!' I suppose that's a slight improvement because I'm sure before the entire game would crash previously. This time I had to manually stop the game as the frozen scene appeared to stay there indefinitely.

Before anyone asks me to try again with Experimental, I'm not interested in running that version of Proton at this point and I'm not playing this game at the moment either, was just seeing if the recent 9.0-4 update made any difference.

**Comment by strikerJava** [comment]
<strike>HI, Just wanted to chime in here, if this isn't the right place for this happy to move the comment where it needs to go. 

Final Fantasy X inside this collection would crash to desktop when launched on my steam deck and this has been an ongoing issue for me no matter what version of proton I would test with on the steam deck. Proton 10.0-1 (beta) seems to have resolved this crash and the game at least boots. I'll need to play the game a bit to test further and see if I see any of the crashes others saw. </strike>
Update: I woke up to start playing it and it appears to be crashing again. Please disregard 

Thanks for the work y'all are doing

**Comment by Wedge009** [comment]
Testing Proton 10.0-3, The Eternal Calm story played through in its entirety without crashes. So seems like the crashing issues have largely been resolved.

**Comment by gort818** [comment]
Seems the latest experimental proton has broken controller support for the launcher.

**Comment by gort818** [comment]
Just tested latest bleeding edge today , controller support working again.

**Comment by noobusinglinux** [comment]
**Symptoms & Reproduction**
The application abruptly exits after approximately 6 hours of continuous gameplay. The terminal output indicates the 32-bit engine is hitting its 2GB memory allocation limit (Allocation 2g limit enabled) rather than a sudden hardware crash or GNOME display freeze.

[steam-359870.log](https://github.com/user-attachments/files/31679539/steam-359870.log)

**Proton Version:** 1787725910 experimental-11.0-20260826-x86_64

<details><summary>Terminal Output1:</summary>

```
~$ grep -i "err:" ~/steam-359870.log | tail -n 30
  0.000:0064:0068:err:module:hacks_init Allocation 2g limit enabled.
  0.000:007c:0080:err:module:hacks_init Allocation 2g limit enabled.
  0.000:0098:009c:err:module:hacks_init Allocation 2g limit enabled.
15082.742:0098:00a4:err:ntoskrnl:ZwLoadDriver failed to create driver L"\\Registry\\Machine\\System\\CurrentControlSet\\Services\\winebth": c0000142
  0.000:00dc:00e0:err:module:hacks_init Allocation 2g limit enabled.
  0.000:00f4:00f8:err:module:hacks_init Allocation 2g limit enabled.
  0.000:011c:0120:err:module:hacks_init Allocation 2g limit enabled.
  0.000:0124:0128:err:module:hacks_init Allocation 2g limit enabled.
err:   OpenVR: could not query value, status 2
15084.511:0124:0128:err:openxr:get_vulkan_extensions Could not query value, status 0x2.
15084.524:0020:0144:err:vrclient:initialize_vr_data Failed to initialize VR info.
  0.000:0148:014c:err:module:hacks_init Allocation 2g limit enabled.
15090.858:0148:014c:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
15090.963:0148:014c:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
15090.964:0148:014c:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
15095.296:0148:014c:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
15095.297:0148:014c:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
  0.000:0204:0208:err:module:hacks_init Allocation 2g limit enabled.
15106.391:0204:0208:err:wgl:X11DRV_WineGL_InitOpenglInfo  couldn't initialize OpenGL, expect problems
15106.417:0124:0194:err:ole:ifproxy_release_public_refs IRemUnknown_RemRelease failed with error 0x800706ba
15106.418:0124:0194:err:ole:ifproxy_release_public_refs IRemUnknown_RemRelease failed with error 0x800706ba
  0.000:02d0:02d4:err:module:hacks_init Allocation 2g limit enabled.
15132.253:02d0:02d4:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
15132.327:02d0:02d4:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
15132.328:02d0:02d4:err:ole:marshal_object Failed to create an IRpcStubBuffer from IPSFactory for {e89f726e-c4f4-4c19-bb19-b647d7fa8478} with error 0x80004002
  0.000:0348:034c:err:module:hacks_init Allocation 2g limit enabled.
15135.065:0348:034c:err:wgl:X11DRV_WineGL_InitOpenglInfo  couldn't initialize OpenGL, expect problems
21457.240:0124:02fc:err:ole:ifproxy_release_public_refs IRemUnknown_RemRelease failed with error 0x800706ba
21457.242:0124:02fc:err:ole:ifproxy_release_public_refs IRemUnknown_RemRelease failed with error 0x800706ba
21457.318:0124:0150:err:service:device_notify_proc failed to get event, error 1726
```

</details><details><summary>Terminal output2:</summary>

```
~$ tail -n 50 ~/steam-359870.log
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x9: pushq %r12
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x7: pushq %r13
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x5: pushq %r14
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x3: pushq %r15
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x1: pushq %rbp
21457.319:0124:0150:trace:seh:RtlUnwindEx calling TEB handler 00006FFFFF8E8150 (rec=0000000003FDE440, frame=0000000003FDE670 context=0000000003FDD060, dispatch=0000000003FDCA10)
21457.319:0124:0150:trace:seh:RtlUnwindEx handler at 00006FFFFF8E8150 returned 1
21457.319:0124:0150:trace:unwind:RtlVirtualUnwind2 type 2 base 6fffff8b0000 rip 6fffff8d4e92 rva 24e92 rsp 3fdeff0
21457.319:0124:0150:trace:unwind:dump_unwind_info **** func 24820-24f7c
21457.319:0124:0150:trace:unwind:dump_unwind_info unwind info at 00006FFFFF92D3F8 flags 0 prolog 0x12 bytes function 00006FFFFF8D4820-00006FFFFF8D4F7C
21457.319:0124:0150:trace:unwind:dump_unwind_info     frame register rbp offset 0x80(%rsp)
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x12: leaq 0x80(%rsp),rbp
21457.319:0124:0150:trace:unwind:dump_unwind_info     0xa: subq $0xa80,%rsp
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x3: pushq %rbx
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x2: pushq %rsi
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x1: pushq %rbp
21457.319:0124:0150:trace:unwind:RtlVirtualUnwind2 type 2 base 6fffff8b0000 rip 6fffff8b66ab rva 66ab rsp 3fdfa90
21457.319:0124:0150:trace:unwind:dump_unwind_info **** func 6690-66b0
21457.319:0124:0150:trace:unwind:dump_unwind_info unwind info at 00006FFFFF92D2BC flags 0 prolog 0x4 bytes function 00006FFFFF8B6690-00006FFFFF8B66B0
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x4: subq $0x28,%rsp
21457.319:0124:0150:trace:unwind:RtlVirtualUnwind2 inside epilog.
21457.319:0124:0150:trace:unwind:RtlVirtualUnwind2 type 2 base 6fffffaa0000 rip 6fffffaa97e8 rva 97e8 rsp 3fdfac0
21457.319:0124:0150:trace:unwind:dump_unwind_info **** func 9370-99c4
21457.319:0124:0150:trace:unwind:dump_unwind_info unwind info at 00006FFFFFABB934 flags 0 prolog 0x1b bytes function 00006FFFFFAA9370-00006FFFFFAA99C4
21457.319:0124:0150:trace:unwind:dump_unwind_info     frame register rbp offset 0x80(%rsp)
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x1b: leaq 0x80(%rsp),rbp
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x13: subq $0x478,%rsp
21457.319:0124:0150:trace:unwind:dump_unwind_info     0xc: pushq %rbx
21457.319:0124:0150:trace:unwind:dump_unwind_info     0xb: pushq %rsi
21457.319:0124:0150:trace:unwind:dump_unwind_info     0xa: pushq %rdi
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x9: pushq %r12
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x7: pushq %r13
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x5: pushq %r14
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x3: pushq %r15
21457.319:0124:0150:trace:unwind:dump_unwind_info     0x1: pushq %rbp
21457.319:0124:0150:trace:seh:RtlRestoreContext returning to 00006FFFFFAA325A stack 0000000003FDFAC0
21457.320:0124:0150:trace:mscoree:DllMain (00006FFFFDA80000, 3, 0000000000000000)
21457.322:0124:0f44:trace:mscoree:DllMain (00006FFFFDA80000, 3, 0000000000000000)
21457.322:0124:0168:trace:unwind:RtlGrowFunctionTable 000000000546E490, 991
21457.322:0124:0f3c:trace:mscoree:DllMain (00006FFFFDA80000, 3, 0000000000000000)
21457.322:0124:0f1c:trace:mscoree:DllMain (00006FFFFDA80000, 3, 0000000000000000)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000004E00000, 0, 0000000000000001)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000004790000, 0, 0000000000000001)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000004740000, 0, 0000000000000001)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000003DC0000, 0, 0000000000000001)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000003DA0000, 0, 0000000000000001)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000010000000, 0, 0000000000000001)
21457.369:0124:0128:trace:mscoree:_CorDllMain (0000000002F70000, 0, 0000000000000001)
21457.370:0124:0128:trace:mscoree:DllMain (00006FFFFDA80000, 0, 0000000000000001)
pid 49769 != 49764, skipping destruction (fork without exec?)
```

</details>

<details><summary>**System Information**</summary>

```
**Computer Information:**
Manufacturer: LENOVO
Model: 30BC
Form Factor: Desktop
No Touch Input Detected
**Processor Information:**
CPU Vendor: GenuineIntel
CPU Brand: Intel(R) Core(TM) i5-6500 CPU @ 3.20GHz
CPU Family: 0x6
CPU Model: 0x5e
CPU Stepping: 0x3
CPU Type: 0x0
Speed: 3600 MHz
4 logical processors
4 physical processors
Hyper-threading: Unsupported
FCMOV: Supported
SSE2: Supported
SSE3: Supported
SSSE3: Supported
SSE4a: Unsupported
SSE41: Supported
SSE42: Supported
AES: Supported
AVX: Supported
AVX2: Supported
AVX512F: Unsupported
AVX512PF: Unsupported
AVX512ER: Unsupported
AVX512CD: Unsupported
AVX512VNNI: Unsupported
SHA: Unsupported
CMPXCHG16B: Supported
LAHF/SAHF: Supported
PrefetchW: Unsupported
BMI1: Supported
BMI2: Supported
F16C: Supported
FMA: Supported
**Operating System:**
Ubuntu 26.04.1 LTS (64 bit)
Kernel Name: Linux
Kernel Version: 7.0.0-30-generic
X Server Vendor: The X.Org Foundation
X Server Release: 12401010
X Window Manager: GNOME Shell
Steam Runtime Version: steam-runtime_1.0.20260618.246542
**Client Information:**
Version: 1785799196
Browser GPU Acceleration Status: Enabled
Browser Canvas: Enabled
Browser Canvas out-of-process rasterization: Enabled
Browser Direct Rendering Display Compositor: Disabled
Browser Compositing: Enabled
Browser Multiple Raster Threads: Enabled
Browser OpenGL: Enabled
Browser Rasterization: Enabled
Browser Raw Draw: Disabled
Browser Skia Graphite: Disabled
Browser Video Decode: Enabled
Browser Video Encode: Disabled
Browser Vulkan: Disabled
Browser WebGL: Enabled
Browser WebGL2: Enabled
Browser WebGPU: Disabled
Browser WebNN: Disabled
**Video Card:**
Driver: Mesa llvmpipe (LLVM 21.1.8, 256 bits)
Driver Version: 4.5 (Compatibility Profile) Mesa 26.0.8-1ubuntu0.3
OpenGL Version: 4.5
Desktop Color Depth: 24 bits per pixel
Monitor Refresh Rate: 59 Hz
VendorID: 0x10de
DeviceID: 0x1c81
Revision Not Detected
Number of Monitors: 1
Number of Logical Video Cards: 1
Primary Display Resolution: 1680 x 1050
Desktop Resolution: 1680 x 1050
Primary Display Size: 18.50" x 11.81" (21.93" diag), 47.0cm x 30.0cm (55.7cm diag)
Primary VRAM: 2944 MB
**Sound card:**
Audio device: Realtek ALC662 rev3
**Memory:**
RAM: 30997 Mb
**VR Hardware:**
VR Headset: None detected
No VR configuration with trackers detected
Most number of trackers in a session: 0
**Miscellaneous:**
UI Language: English
LANG: en_US.UTF-8
Total Hard Disk Space Available: 3751351 MB
Largest Free Hard Disk Block: 3121301 MB
**Storage:**
Number of SSDs: 2
SSD sizes: 4000G,0B
Number of HDDs: 0
Number of removable drives: 0
```

</details>

**Comment by kisak-valve** [comment]
Hello @noobusinglinux, something's off with your NVIDIA driver install. The NVIDIA proprietary driver requires that the driver's running kernel module, 32 bit userspace libraries, and 64 bit userspace libraries are all installed and exactly the same version to work properly. Your Proton log and system information from Steam both indicate that the 32 bit userspace libraries for that driver are missing and Steam's being forced to fail-through to LLVMpipe (Mesa's faster CPU renderer) to run at all.

I'd check `libnvidia-gl-580:i386` with something like `apt policy libnvidia-gl-580 libnvidia-gl-580:i386`, and install it if it isn't already with `sudo apt install libnvidia-gl-580:i386`.
